indico/modules/events/registration/fields/simple.py

```diff
@@ -388,3 +389,24 @@ def _indico_email(value):
                 raise ValidationError(_('Invalid email address'))
 
         return _indico_email
+
+
+class PictureField(FileField):
+    name = 'picture'
+    setup_schema_fields = {
+        'min_picture_size': fields.Integer(load_default=0, validate=validate.Range(0, 1200)),
+    }
+
+    def get_validators(self, existing_registration):
+        def _picture_size_and_type(value):
+            min_picture_size = self.form_item.data.get('min_picture_size') or None
+            file = File.query.filter(File.uuid == value, ~File.claimed).first() if value else None
+            if file and file.content_type not in ('image/png', 'image/jpg', 'image/gif'):
```

Comment:
Yes it can be included. I can include it in the accepted file types for the drop zone as well. Although because the images go through the cropper and that one produces a png image only the png image makes it to the serverside validator